editor/js/Codemirror.js

```diff
@@ -1,11 +1,10 @@
 import { basicSetup } from 'codemirror';
 import { EditorView, keymap } from '@codemirror/view';
-import { javascript, esLint } from '@codemirror/lang-javascript';
+import { javascript } from '@codemirror/lang-javascript';
 import { EditorState } from '@codemirror/state';
 import { oneDark } from '@codemirror/theme-one-dark';
-import { lintGutter, linter } from '@codemirror/lint';
+import { lintGutter } from '@codemirror/lint';
```

Comment:
This looks fine, but maybe let's add a comment for the future to explain why linter is not used (too large in frontend bundle) and avoid re-adding this in the future